### PR TITLE
Explore page

### PR DESCRIPTION
**Pull request by LayaTaal**
Updates to the Explore the Data page to incorporate filtering.

### CODE REVIEW

File: components/charts/chartsjs/DoughnutChart.js [hunk 1]
```diff
@@ -70,50 +70,12 @@ const transformData = (name, meta, data) => {
 
   // Calculate the total # of incidents per data type
   // We are no longer removing values of 0, or negative numbers, since these have meaning in some cases
-  const dataTotal = meta.map((metaValue, index) => filterItems(data, index).length);
-
-  // Age records need to be handled uniquely, otherwise all other data is just being grouped as it is.
-  if (name !== 'age_at_time_of_death') {
-    meta.forEach((lookup, index) => {
```
Comment (judan) [inline components/charts/chartsjs/DoughnutChart.js]: Hey @LayaTaal, were lookups going to go away in the proposed database changes? This language is being used throughout. But maybe it will change later? If it's not, then can we use different phrasing here, like "record(s)" or "key(s)"?

Comment (LayaTaal) [reply components/charts/chartsjs/DoughnutChart.js]: Thanks @judan I kept the previous language in place to not make too drastic of a shift, but I am happy to reword this to be me clear. I will update this morning.
